contracts/PermissionlessNodeRegistry.sol

```diff
@@ -381,6 +394,19 @@ contract PermissionlessNodeRegistry is
         emit UpdatedStaderConfig(_staderConfig);
     }
 
+    /**
+     * @notice set the cached non-terminal keys count for an operator
+     * @dev only `MANAGER` role can call, used to initialize the cached count
+     * @param _operatorId Id of the operator
+     * @param _nonTerminalKeysCount the non-terminal keys count to set
+     */
+    function setOperatorNonTerminalKeysCount(uint256 _operatorId, uint64 _nonTerminalKeysCount) external {
+        UtilLib.onlyManagerRole(msg.sender, staderConfig);
```

Comment:
do we need checks here to ensure no validator of this operator is in deposit queue etc?